hrjobcontract/js/src/job-contract/controllers/contract-list.controller.js

```diff
@@ -58,75 +53,108 @@ define([
         'of the contract.</p>' +
         '</div>')
     };
+    $scope.utils = {
+      contractListLen: contractList.length
+    };
 
-    for (entityName in entityServices) {
-      promiseFields[entityName] = entityServices[entityName].getFields();
-    }
-
-    $q.all(promiseFields).then(function (fields) {
-      $scope.fields = fields;
-
-      $log.debug('FIELDS:');
-      $log.debug(fields);
+    $scope.delete = deleteContract;
+    $scope.modalContract = modalContract;
+    $scope.toggleIsPrimary = toggleIsPrimary;
 
+    (function init () {
       for (entityName in entityServices) {
-        promiseModel[entityName] = entityServices[entityName].model(fields[entityName]);
+        promiseFields[entityName] = entityServices[entityName].getFields();
       }
 
-      return $q.all(promiseModel);
-    }).then(function (model) {
-      $scope.model = model;
+      $q.all(promiseFields).then(function (fields) {
+        $scope.fields = fields;
```

Comment:
Can you please explain in the PR description why this was renamed from `models` to `fields`?